src/app/solutions/belus.tsx

```diff
@@ -218,9 +218,13 @@ export const BelusChallenge: React.FC<BelusChallengeProps> = ({}) => {
 
   return (
     <Animated.View style={[styles.flexible, bgStyle]}>
+      <Stack.Screen options={{ title: "Belu's Challenge" }} />
       <Animated.View style={[backgroundStyle, styles.background]} />
       <Animated.View style={fruitStyle}>
-        <Image source={Avocado} style={styles.fruit} />
+        <Image
+          source={require('../../../assets/images/food/avocado.png')}
```

Comment:
Just curious, why use `require` instead of `import`?

Reply:
Good catch! Had some glitch on this, but it's working now - I was probably missing some path changes. Reverted!